### ISSUE TITLE
🧩 Feature: Drag Select

### ISSUE THREAD

**Issue by KL13NT** [KL13NT/clippy#36]
**Is your feature request related to a problem? Please describe.**
On multiple operating systems, when clicking, holding, and dragging the mouse over a collection of files, all the files in the selection range are selected. If this could be implemented in Clippy it'd make the effort of selecting multiple consecutive entry items a lot easier. 

**Describe the solution you'd like**
Adding a `click` handler that adds a `mousemove` listener and creates a rectangular selection zone based on mouse movement may be a way to implement this.

**Describe alternatives you've considered**
A library exists for this, called [SelectionJS](https://simonwep.github.io/selection/). We could either mock its behavior or use it right away. It's only 4 KBs.

**Additional context**
The behavior would be the same across Windows and Linux since it's rather dependent on the render Chromium process.

**Comment by bywasik** [comment]
Hi. Can you assign me to this one?

**Comment by KL13NT** [comment]
Absolutely. Note that using the library inside of React directly could break existing functionality because of the classes added by the library, so you should try that out first before going all-in. The affected 'selectable' target will be the `ul`.
